packages/engine/src/audio/components/PositionalAudioHelperComponent.ts

```diff
@@ -0,0 +1,153 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/EtherealEngine/etherealengine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Ethereal Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Ethereal Engine team.
+
+All portions of the code written by the Ethereal Engine team are Copyright © 2021-2023 
+Ethereal Engine. All Rights Reserved.
+*/
+
+import { useDidMount } from '@etherealengine/common/src/utils/useDidMount'
+import { Entity, defineComponent, useComponent, useEntityContext } from '@etherealengine/ecs'
+import { NO_PROXY } from '@etherealengine/hyperflux'
+import { useHelperEntity } from '@etherealengine/spatial/src/common/debug/DebugComponentUtils'
+import { useDisposable, useResource } from '@etherealengine/spatial/src/resources/resourceHooks'
+import { useEffect } from 'react'
+import { BufferAttribute, BufferGeometry, InterleavedBufferAttribute, Line, LineBasicMaterial, MathUtils } from 'three'
+import { AudioNodeGroup } from '../../scene/components/MediaComponent'
+
+export const PositionalAudioHelperComponent = defineComponent({
+  name: 'PositionalAudioHelperComponent',
+
+  onInit: (entity) => {
+    return {
+      name: 'positional-audio-helper',
```

Comment:
is this property necessary to be configurable? does this name field do anything functional? if not, we should just hardcode it and not expose it as an API

Reply:
I just forgot to set it in the component, I've tried to have all the debug components reflect the names of the components that they're tied to, ie. an entity named MyAudio with a PositionalAudioComponent will have it's helper entity named MyAudio-positional-audio-helper. I could update the api to get the name automatically from the name component in the helper, but some components make multiple helpers so having it configurable allows having an index in the name